x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.ts

```diff
@@ -191,7 +179,7 @@ export const OverviewLogic = kea({
       (isFederatedAuth: boolean) => (isFederatedAuth ? 'halves' : 'fourths'),
```

Comment:
If you tried to change this to return a type that doesn't match what's defined for the value, you'll also now get a type error, e.g.

```suggestion
      (isFederatedAuth: boolean) => false,
```